src/components/roomResults/RoomCardModal/PromotionCard/PromotionCard.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box, Button, Typography } from "@mui/material";
+import React from "react";
+import { useAppSelector } from "../../../../redux/store";
+
+type Props = {};
+
+const PromotionCard = (props: Props) => {
+  const selectedCurrency = useAppSelector((state) => state.currency.selectedCurrency);
+
+  return (
+    <Box
+      mt={"1rem"}
+      sx={{
+        display: { md: "grid", sm: "block" },
+        gridTemplateColumns: "70% 1fr",
+        boxShadow: "0px 2px 2px rgba(0, 0, 0, 0.1)",
+        border: "1px solid #EFF0F1",
+        maxWidth: { lg: "40vw", md: "60vw" },
```

Comment:
can we extract the inline SX styles for all the components thoughout the code and store them in variables and then pass them sx
ex : 
const promoCardStyle = {
      display: { md: "grid", sm: "block" },
        gridTemplateColumns: "70% 1fr",
        boxShadow: "0px 2px 2px rgba(0, 0, 0, 0.1)",
        border: "1px solid #EFF0F1",
        maxWidth: { lg: "40vw", md: "60vw" },
        }
        then, 
        <Box sx={promoCardStyle} >
        This refactoring would make your components structured and clean and easy to understand.